CMakeLists.txt

```diff
@@ -14,23 +14,12 @@ endif()
 include(vcpkg OPTIONAL)
 
 find_package(UnitTest++ CONFIG REQUIRED)
-if((NOT DEFINED UnitTest++_INCLUDE_DIRS) AND (DEFINED UTPP_INCLUDE_DIRS))
-set(UnitTest++_INCLUDE_DIRS  "${UTPP_INCLUDE_DIRS}")
-## have corrected the UnitTest++Config.cmake file to point UnitTest++_INCLUDE_DIRS to the correct path for the headers
-endif()
+find_path(UnitTest++_INCLUDE_DIRS "UnitTest++/UnitTest++.h")
```

Comment:
I'm not sure about the find path. Hopefully the find_package should have populated the UnitTest++_INCLUDE_DIRS variable, and overwriting it might not be the best idea. I'll look at this some more.